kani-compiler/src/args.rs

```diff
@@ -88,6 +90,12 @@ pub struct Arguments {
     /// Print the final LLBC file to stdout.
     #[clap(long)]
     pub print_llbc: bool,
+    /// If we are running the autoharness subcommand, the functions to autoharness
+    #[arg(long = "autoharness-include-function", num_args(1))]
+    pub autoharness_included_functions: Vec<String>,
+    /// If we are running the autoharness subcommand, the functions to exclude from autoverification
+    #[arg(long = "autoharness-exclude-function", num_args(1))]
+    pub autoharness_excluded_functions: Vec<String>,
```

Comment:
Another option we can consider adding (perhaps in a follow-up PR) is one to only apply autoharness to public functions in a crate. Non-public functions often have pre-conditions, and may fail without constraints on the symbolic values.